public/app/common/components/icon/style.scss

```diff
@@ -29,6 +29,10 @@
     background-image: url('./images/state-disconnected.svg');
```

Comment:
You renamed _state_ to _status_. +1 for consistent naming. You might want to rename images also.